### PR TITLE
About page with some styling

### PR DESCRIPTION
**Pull request by alodahl**
image
image

### CODE REVIEW

File: web/src/components/View/View.css [hunk 1]
```diff
@@ -19,8 +19,17 @@
 
 .view-container__title {
     color: var(--colarkfontcolor);
+<<<<<<< HEAD
```
Comment (lianthompson) [inline web/src/components/View/View.css]: do we want to keep the <<< HEAD ===== >>>>> lines

File: web/src/components/team-member/team-member.css [hunk 2]
```diff
@@ -17,35 +16,23 @@ img.team-member__photo, .team__accent-dot {
...
 .ellie-photo {
-  top: 40px;
-  left: 240px;
+  top: 140px;
+  left: 140px;
 }
 
 .lian-photo {
-  bottom: 20px;
+  /* bottom: 20px; */
```
Comment (lianthompson) [inline web/src/components/team-member/team-member.css]: is this going to be uncommented out later?

File: web/src/components/View/View.css (hunk 1 above)
Comment (alodahl) [reply web/src/components/View/View.css]: nope!

File: web/src/components/team-member/team-member.css (hunk 2 above)
Comment (alodahl) [reply web/src/components/team-member/team-member.css]: that was an accident.  I just made a PR with what I had, but i didnt have time to polish it.

**Pull request merged**